OrbitGl/TrackManager.cpp

```diff
@@ -252,8 +252,8 @@ void TrackManager::UpdateMovingTrackSorting() {
     int moving_track_current_position = -1;
     for (auto track_it = visible_tracks_.begin(); track_it != visible_tracks_.end(); ++track_it) {
       if (moving_track->GetPos()[1] >= (*track_it)->GetPos()[1]) {
-        visible_tracks_.insert(track_it, moving_track);
```

Comment:
I already LGTM'd this, but afterwards, I was just wondering: Wouldn't it be clearer to make this

```
auto inserted_it = visible_tracks_.insert(track_it, moving_track);
moving_track_current_position = inserted_it - visible_tracks_.begin();
```

Reply:
Yep lgtm, I made the change, thanks!